### DISCUSSION TITLE
Meter Ethereum Hackathon

### DISCUSSION THREAD

**Discussion by Fankouzu** [NonceGeek/Move-Camp#49] [Announcements]
Build a dApp on Meter.io for a shot at some of the $100,000 in $MTRG prizes.
Who can participate

Above legal age of majority in country of residence

All countries/territories, excluding standard exceptions 

Deadline: Apr 11, 2022 @ 5:00pm EDT

[more detial](https://meter-ethereum.devpost.com/)

Meter.io is a highly decentralized Et[here](https://meter.io/developer-program.html)um scaling solution with a built-in metastable gas currency. It connects to Ethereum and other blockchains as a layer-two protocol and allows smart contracts to scale and communicate seamlessly through heterogeneous blockchain networks.

# Meter Hackathon's goal is to onboard new quality projects (DApps, DeFi, DAOs, DEXs) into Meter Ecosystem. 

Existing Ethereum dApps can be migrated to Meter with almost no changes! This allows Meter to be able to supercharge Ethereum and run at 1500TPS per chain. Built on the most advanced consensus algorithm Hotstuff, our platform advantages include:

1) We are Ethereum RPC compatible with unique features like front running/MEV resistant, low gas cost in metastable coin, high throughput and instant finality.

2) Interoperability with multiple blockchains.

3) Highly decentralized validators to ensure censorship resistance and security.

Read more about our developer program here. We invite the top dApps in each category to apply to our [grant program](https://medium.com/meter-io/meter-developer-grant-program-b1596cc9fe88) for funding support post Hackathon. In addition, we offer all developers in the grant program full marketing and technical support.

# What to Build
Build a working prototype on Meter.io within one of the Categories. Your project can be brand new or a migrated Ethereum dApp new to Meter.io.

Categories:

- DeFi: This includes stable coin DEXes, IDO platform, Lending borrowing, Synthetic assets, OHM like protocol owned liquidity, stablecoins and other DeFi applications.
- Gaming dApp: This includes Play to Earn, metaverse, gamesFi applications.
- NFT dApp: Create Dapps, Tools, and platforms for NFTs creating/minting and trading. Create unique NFT art collections or games.
- Web3 Integration: Create other web3 tools and infrastructure to facilitate application development on Meter, for example storage layer, DAO, tools and platform (Oracle integration, Uniswap, Saddle, Opensea, and etc.)
# REQUIREMENTS
# What to Submit
- Provide a URL to your open-source code repository.
- Provide a URL and instructions to access a functional demo of your project free of charge. Your project must run on an iOS, Android, or Web (mobile or desktop) platform.
- Include a video (about 3 minutes) that demonstrates your submission. Videos must be uploaded to YouTube, Vimeo, or Facebook Video and made public.
# PRIZES
$100,000 in prizes

 Grand Prize
$30,000 in $MTRG

 1st Place DeFi
$10,000 in $MTRG

 2nd Place DeFi
$7,500 in $MTRG

 3rd Place Defi
$5,000 in $MTRG

 1st Place Gaming
$10,000 in $MTRG

 2nd Place Gaming
$7,500 in $MTRG

 3rd Place Gaming
$5,000 in $MTRG

 1st Place NFT dApp
$10,000 in $MTRG

 2nd Place NFT dApp
$7,500 in $MTRG

 3rd Place NFT dApp
$5,000 in $MTRG

 Best web3 integration
$2,500 in $MTRG

**Comment by leeduckgo** [comment]
GooD!
